src/frontend/helpers/index.ts

```diff
@@ -148,6 +149,41 @@ export function getPlatformName(platform: string): string {
   }
 }
 
+const getLastPartOfUrl = (url: string) => {
+  return url.split('/').pop()
+}
+
+export const fetchEpicListing = async (projectId: string) => {
```

Comment:
we'll just need to remember to qa this for new epic game forwarder listings since developers could enter an epic url that might break this

Reply:
Yes. The only way to make this 100% reliable would be to have the Epic `appName` in the dev portal. For every new listing with quests we will need to double check this and even adjust it.